public/javascripts/main.js

```diff
@@ -178,6 +178,17 @@ $(function() {
             }
         }
     });
+
+    $(".unsafe-link-back").click(function () {
+        window.history.back();
+    });
+
+    $(".unsafe-link-continue").click(function () {
+        var remote = $(".unsafe-link-continue").data("remote");
+
+        location.replace(location.href);
```

Comment:
Is this line needed?

Reply:
Yes. 

External Link -> Link out page -> User clicks continue..
User wants to go back in browser:
Without this code he would go to the link out page.
With this code he is going back to the page where he clicked the link on.